### ISSUE TITLE
Comparative dscriptions seem inconsistent

### ISSUE THREAD

**Issue by terracoda** [phetsims/quadrilateral#262]
In the following example, Corner D is describe as about half as wide as its opposite corner (Corner B). But then when I move over to Corner B, it is described as "right angle, far wider than opposite corner, smaller than adjacent corners".

Why isn't Corner B "about twice as wide as opposite corner" (Corner D)?

Just wondering if we have a bug somewhere in the ranges for defined for the qualitative comparative sizes??
<img width="670" alt="image" src="https://user-images.githubusercontent.com/1902116/203624518-99a017f0-6938-4119-863c-59fe526a5d59.png">

**Comment by jessegreenberg** [comment]
0.729 * 2 = 1.458 -> (1.571 - 1.458) = 0.113 -> way bigger than `interAngleToleranceInterval`

1.571 / 2 = 0.7855 -> 0.7855 - 0.729 = 0.056 -> way bigger than `interAngleToleranceInterval`

So my understanding is that corner B description is correct and corner D description is buggy.

**Comment by jessegreenberg** [comment]
Aha, it is because after checking `interAngleToleranceInterval`, we fall back to this map:

```
// If ratio of an angle to another is within this range it is 'about half as large as the other'.
const ABOUT_HALF_RANGE = new Range( 0.4, 0.6 );

// If ratio of angle to another is within this range it is 'about twice as large as the other'.
const ABOUT_TWICE_RANGE = new Range( 1.9, 2.1 );

// Maps the difference in angles between two vertices to a description string.
const angleComparisonDescriptionMap = new Map<Range, string>();
angleComparisonDescriptionMap.set( new Range( 0, 0.1 ), farSmallerThanString );
angleComparisonDescriptionMap.set( new Range( 0.1, 0.4 ), muchSmallerThanString );
angleComparisonDescriptionMap.set( ABOUT_HALF_RANGE, aboutHalfAsWideAsString );
angleComparisonDescriptionMap.set( new Range( 0.6, 0.8 ), aLittleSmallerThanString );
angleComparisonDescriptionMap.set( new Range( 0.8, 1 ), similarButSmallerThanString );
angleComparisonDescriptionMap.set( new Range( 1, 1.3 ), similarButWiderThanString );
angleComparisonDescriptionMap.set( new Range( 1.3, 1.6 ), aLittleWiderThanString );
angleComparisonDescriptionMap.set( new Range( 1.6, 1.9 ), muchWiderThanString );
angleComparisonDescriptionMap.set( ABOUT_TWICE_RANGE, aboutTwiceAsWideAsString );
angleComparisonDescriptionMap.set( new Range( 2.1, Number.POSITIVE_INFINITY ), farWiderThanString );
```

1.571 / 0.729 = 2.155 -> ***larger than 2.1 so we NOT describe as "about twice as large"***

0.729 / 1.571 -> 0.464 -> ***greater than 0.4 so we DO describe as "about half as large"***

We cannot guarantee that the ratios will be symmetric. @terracoda or @BLFiedler how would you like to handle this?

**Comment by terracoda** [comment]
Here's a screenshot where Corner D is "half as wide" and Corner B is "twice as wide".
<img width="670" alt="image" src="https://user-images.githubusercontent.com/1902116/203635508-1e8ebec6-da29-4512-b991-bb973e2accf1.png">

So in this case it works well, maybe because B is exactly twice as wide as D.

**Comment by jessegreenberg** [comment]
Yep, the ratios have to be within these ranges which is the case for https://github.com/phetsims/quadrilateral/issues/262#issuecomment-1325593018, but not for the original screenshot.
```js
// If ratio of an angle to another is within this range it is 'about half as large as the other'.
const ABOUT_HALF_RANGE = new Range( 0.4, 0.6 );

// If ratio of angle to another is within this range it is 'about twice as large as the other'.
const ABOUT_TWICE_RANGE = new Range( 1.9, 2.1 );
```

**Comment by terracoda** [comment]
I guess you can't compare the number of slices? What a pain

**Comment by terracoda** [comment]
I feel like the information is not very consistent. 
Does it have to be a ratio?

**Comment by jessegreenberg** [comment]
We decided to use ratios (percentages) in https://github.com/phetsims/quadrilateral/issues/219. At one point we were using raw angles (slices) to generate these descriptions. I can't remember why we switched, but maybe it had its own problems.

**Comment by terracoda** [comment]
@BLFiedler, do you think it will do any good to adjust the range?

**Comment by jessegreenberg** [comment]
A potential brute force method could be to only describe opposite/adjacent corners as half/twice the other if both ratios are within the ranges of https://github.com/phetsims/quadrilateral/issues/262#issuecomment-1325614060. Or could describe both as half/twice if either is within the range.

**Comment by BLFiedler** [comment]
We should do this for both Angle and Length comparisons for "about twice as"

**Comment by terracoda** [comment]
In today's meeting we settled on doubling the range for the about twice as wide as and about twice and long as regions

Corner comparisons
![image](https://user-images.githubusercontent.com/1902116/204615369-067d47a2-6c72-43a6-b44f-35dd833309f6.png)

Side comparisons
![image](https://user-images.githubusercontent.com/1902116/204615500-1e62023e-db77-4ff2-8066-1619220303de.png)

**Comment by terracoda** [comment]
@jessegreenberg, did you already implement this while were discussing it? If so, please re-assign to me.

**Comment by jessegreenberg** [comment]
OK, this was done in the above commit. Back to you @terracoda.

**Comment by terracoda** [comment]
The scaled adjustment sounds way better! Thanks @jessegreenberg  and @BLFiedler.
